manifests/init.pp

```diff
@@ -75,7 +75,7 @@
   if ($manage_default_ca_cert) and ($facts['os']['family'] == 'RedHat') {
     exec { 'download_install_katello_cert_rpm':
       path    => ['/usr/bin', '/bin',],
-      command => "curl -k -o /tmp/katello-ca-consumer-latest.noarch.rpm ${satellite_url}/pub/katello-ca-consumer-latest.noarch.rpm && yum -y install /tmp/katello-ca-consumer-latest.noarch.rpm", # rubocop:disable Layout/LineLength
+      command => ['curl',  '-k',  '-o', '/tmp/katello-ca-consumer-latest.noarch.rpm', "${satellite_url}/pub/katello-ca-consumer-latest.noarch.rpm", '&&', 'yum', '-y', 'install', '/tmp/katello-ca-consumer-latest.noarch.rpm'], # rubocop:disable Layout/LineLength
```

Comment:
This isn't valid because it uses shell (see the `&&`). It can't be safely done this way.

You can split it in a `file` resource with `${satellite_url}/pub/katello-ca-consumer-latest.noarch.rpm"` as the source and then notify a package resource, but you'll always have some intermediate file on disk.